cg/services/file_delivery/deliver_files_service/deliver_files_service_factory.py

```diff
@@ -60,11 +61,13 @@ def __init__(
         hk_api: HousekeeperAPI,
         rsync_service: RsyncAPI,
         tb_service: TrailblazerAPI,
+        analysis_service: AnalysisService,
```

Comment:
Can this analysis service be part of the TrailblazerAPI? Is there a gain in keeping it separated? I think that it is just one dependency more to keep track of but it is just an extension of the TbAPI. But out of scope, feel free to resolve

Reply:
I agree separate PR